optimum/exporters/openvino/model_patcher.py

```diff
@@ -39,6 +41,14 @@ def patch_model_with_bettertransformer(model):
             + COLOR_RESET
         )
 
+    if is_transformers_version(">=", "4.38") and _openvino_version < "2024.1.0-14612":
+        log.warn(
+            COLOR_RED + f"[WARNING] Stateful models are not supported with Transformers {_transformers_version} and "
+            "this OpenVINO version. For good performance, consider using a nightly OpenVINO build: "
```

Comment:
```suggestion
            f"this OpenVINO version {_openvino_version}. For good performance, consider using a nightly OpenVINO build: "
```

Reply:
I didn't do that because if you install 2024.0.0 release, _openvino_version is '2024.0.0-14509' and I didn't think that version with commit-id would be recognizable, and it's not useful for users, because they cannot do `pip install openvino==2024.0.0-15000` for example. But if you want I can change it.

Reply:
I see, we could use `_openvino_version.split("-")[0]` but no strong opinion on this,current integration also works for me

Reply:
Now showing full version if people have installed a prerelease, but only main version for 2024.0.0 or lower installs.